RELEASE_NOTES

```diff
@@ -3,8 +3,77 @@ For more detailed information, please see the git log.
+- various other enhancements, including:
+    - enhance ORCA easyconfig for compatibility with SLURM (#1819)
+    - enable -fPIC in GraphicsMagick easyconfig, required by Octave (#2764)
+    - clean up binutils easyconfigs to use binutils easyblock (#3006)
+    - add include/GraphicsMagick to CPATH in GraphicsMagick easyconfigs (#3034)
+    - update SuiteSparse easyconfigs according to updated SuiteSparse easyblock (#3050)
+- various bug fixes, including:
+    - fix perl extensions download urls   (#2738)
+    - add Autconf as build dep for GCCcore (#2772)
+    - fix versions of extensions in Bioconductor 3.2 bundles (#2769)
+    - fix (build) deps for intel/2016a easyconfigs of cairo, libXext, libXrender (#2785, #2874)
+    - use 'env' wherever preconfig/build/installopts is used to set environmental variables (#2807, #2811, #2812)
+    - add zlib as explicit dep in Tk easyconfigs (#2815)
+    - consistently specify to use -fgnu89-inline flag in M4 1.4.17 easyconfigs (#2774, #2779, #2816)
+    - fix homepage and description in Pygments easyconfigs (#2822)
+    - include pkg-config as build dependencies for libXau, libXdmcp, libxcb (#2827)
+    - consistently use XORG_*_SOURCE constants (#2829, #2830)
+    - update source URLs in ScientificPython easyconfig files (#2847)
+    - use XORG_LIB_SOURCE in libICE/libSM easyconfigs (#2848)
+    - add checksums in SuiteSparse easyconfigs (#2849)
+    - fix build deps for GObject-Introspection (#2852)
+    - correctly specify Perl location in git easyconfig (#2866)
+    - fix bitstring 3.1.3 download URL in Python easyconfigs, source tarball on PyPI disappeared (#2880)
+    - fix Perl dependency in worker, it required non-standard Perl modules (#2884)
+    - add XZ as dependency in Python 3.5.1 easyconfigs, required for lzma (#2887)
+    - fix download URL for packmol (#2902)
+    - drop usempi toolchain in numexpr easyconfigs, not needed (#2937)
+    - fix use of resolve_dependencies in tests according to changes in framework (#2952)
+    - add dependency extensions for MarkupSafe and jsonscheme in IPython 3.2.3 easyconfigs (#2967)
+    - add patch for matplotlib 1.5.1 to fix Tcl/Tk library paths being used (#2971)
+    - add xproto build dependency for makedepend v1.0.5 (#2982)
+    - disable parallel build for Doxygen (#2986)
+    - fix source URLs for FreezeThaw and Tie::Function extensions for Perl v5.22.1 (#2988)
+    - add sed command in worker easyconfig files to fix module_path in conf/worker.conf (#2997, #3000)
+    - drop toolchainopts from Eigen easyconfigs, since it is headers-only (#3025)
+    - clean up dummy bzip2 easyconfig, define buildopts rather than defining $CC and $CFLAGS via os.environ (#3036)
+    - use %(pyshortver)s template rather than hardcoding 2.7 in VTK easyconfigs (#3052)
+    - correct install location of OpenCV Python bindings (#3054)
+    - include XZ as dependency for libunwind (#3055)
```

Comment:
lgtm 

(XZ: and one day we need to run ldd across all modules' binaries and detect how much more fun there is...)